linux_os/guide/system/software/updating/ensure_gpgcheck_globally_activated/rule.yml

```diff
@@ -66,4 +68,4 @@ ocil: |-
     <tt>gpgcheck</tt> line or a setting of <tt>0</tt> indicates that it is
     disabled.
 
-platform: yum
+platform: machine
```

Comment:
The reason this and few other rules for repo gpg have `platform: yum` is to restrict these checks in certain containers that don't have `yum` installed, but not restrict from all containers. Is it the same case with SLE?
If so, a CPE for `zypp` should be added.

In any case, it seems that `rhel` and `sle` will have different `platform`. But I don't think the `platform` tag has any multi-product awareness implemented.
So I guess some Jinja conditionals around `platform` will be needed.

Reply:
Thanks for the suggestion. I agreed that `platform: machine` may be too broad. Originally I was making two assumptions which may not be true for the non-SUSE platforms.

1. that "prodtype" should've restricted the rule to those specific products.
2. package manager is consistent. For the SUSE platforms, all of them are using zypper to manage the packages, container or otherwise. I didn't realize this may not be true for the non-SUSE platforms.

I agree that adding a CPE for zypp is a safer approach.